app/templates/partials/introduction/preview.html

```diff
@@ -18,18 +18,19 @@ <h3 class="venus u-mt-s">{{ content.title }}</h3>
 </div>
 {% if intro.questions %}
 <div class="accordion js-accordion">
-        <div class="mars accordion__controls">
-            <a class="accordion__control js-accordion-open-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Show all" tabindex="0" aria-hidden="true">{{ _("Show all") }}</a>
-            <a class="accordion__control js-accordion-close-all u-hidden" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Hide all" tabindex="0" aria-hidden="true">{{ _("Hide all") }}</a>
+        <div class="accordion__controls">
+            <button class="btn btn--secondary btn--small accordion__control js-accordion-toggle-all u-wa--@xs" data-ga="click" data-ga-category="Preview Survey" data-ga-action="Show all" data-close-all-label="{{ _("Hide all") }}" data-open-all-label="{{ _("Show all") }}" data-open="false" aria-hidden="true">{{ _("Show all") }}</button>
         </div>
-        <dl class="js-accordion-content">
+
+        <div class="js-accordion-content">
             {% for question in intro.questions %}
```

Comment:
I know its existing and you are updating the markup, but can we not use the `partials/accordion.html` here to reduce duplication?